frontend/src/App.js

```diff
@@ -85,7 +85,7 @@ class App extends Component<void, AppState> {
 
   // The history parameter should be the react-router history
   logout = (history: Array<string>) => {
-    document.cookie = 'token=;max-age=0';
+    document.cookie = 'token=;max-age=0;path=/';
```

Comment:
What does this change ?

Reply:
Ok I found the same answer here : https://stackoverflow.com/a/57644049 and it does work, good job !